### PR TITLE
Per ad per day advertiser table/export

### PR DESCRIPTION
**Pull request by davidfischer**
Creates a metabase view for advertisers to view a per ad per day breakdown. It isn't visualized but just a table that they can download and export.

## Screenshot

Screenshot from 2024-10-02 12-43-26

Screenshot from 2024-10-02 12-45-36

### CODE REVIEW

File: adserver/templates/adserver/reports/advertiser.html [hunk 1]
```diff
@@ -44,6 +44,11 @@ <h2 class="col-md-8">{% trans 'Total results for all ads across all flights' %}<
 
         {% if export_url %}
           <aside class="mb-3 col-md-4 text-right">
+            <a href="{% url 'advertiser_per_ad_report' advertiser.slug %}" class="btn btn-sm btn-outline-secondary" role="button" aria-pressed="true">
```
Comment (ericholscher) [inline adserver/templates/adserver/reports/advertiser.html:47]: I wonder about putting it in the sidebar under `Reports` as well? Maybe after we test it a bit more?

File: adserver/templates/adserver/reports/advertiser-per-ad.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+{% block breadcrumbs %}
+  {{ block.super }}
+  <li class="breadcrumb-item active">{% trans 'Per Ad Per Day Report' %}</li>
+{% endblock breadcrumbs %}
+
+
+{% block explainer %}
+<section class="mb-5">
+  <h3>{% trans 'About this report' %}</h3>
+  <p>{% trans 'This report shows a table of data with one entry for each ad for every day that ad is shown. The data can be exported to CSV/Excel for further analysis with the icon at the bottom right.' %}</p>
```
Comment (ericholscher) [inline adserver/templates/adserver/reports/advertiser-per-ad.html:23]: Might want to make the export info a bit more prominent? Perhaps bolded or something?

**Review by ericholscher — APPROVED** [review]
Looks 💯

File: adserver/templates/adserver/reports/advertiser.html (hunk 1 above)
Comment (davidfischer) [reply adserver/templates/adserver/reports/advertiser.html:47]: Sounds good.

File: adserver/templates/adserver/reports/advertiser-per-ad.html (hunk 2 above)
Comment (davidfischer) [reply adserver/templates/adserver/reports/advertiser-per-ad.html]: Done

**Pull request merged**